cmd/updater/update.sh

```diff
@@ -243,29 +241,31 @@ function check_for_updater() {
     if ! curl -sSL "$UPDATER_URL" -o "$UPDATER_ARCHIVE"; then
         echo "failed to download updater archive from ${UPDATER_URL} using curl."
         exit 1
+    else
+        echo "Downloading $UPDATER_URL"
     fi
 
     if [ ! -f "$UPDATER_ARCHIVE" ]; then
         echo "downloaded file ${UPDATER_ARCHIVE} is missing."
         exit
+    else
+        echo "Downloaded into file ${UPDATER_ARCHIVE}"
     fi
 
     # if -verify command line flag is set, try verifying updater archive
     if [ "$VERIFY_UPDATER_ARCHIVE" = "1" ]; then
-        # if linux, check for checksum and signature validation dependencies
+        # check for checksum and signature validation dependencies
         local GPG_VERIFY="0" CHECKSUM_VERIFY="0"
-        if [ "$UNAME" = "Linux" ]; then
-            if type gpg >&/dev/null; then
-                GPG_VERIFY="1"
-            else
-                echo "gpg is not available to perform signature validation."
-            fi
+        if type gpg >&/dev/null; then
+            GPG_VERIFY="1"
+        else
+            echo "gpg is not available to perform signature validation."
+        fi
 
-            if type sha256sum &>/dev/null; then
-                CHECKSUM_VERIFY="1"
-            else
-                echo "sha256sum is not available to perform checksum validation."
-            fi
+        if type sha256sum &>/dev/null; then
```

Comment:
I had issues with this not working on my macos device. I believe `shasum` is installed by default. `sha256sum` would would be required to be installed separately, but considering the latest release mentioned that those tools needed to be installed, maybe it's not a big deal. For instance, `brew install coreutils` would be required as that's how my copy of `sha256sum` was installed.